content/applications/inventory_and_mrp/manufacturing/plm/version_control.rst

```diff
@@ -0,0 +1,142 @@
+===============
+Version control
+===============
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Material)`
+.. |BOMs| replace:: :abbr:`BoMs (Bill of Materials)`
+.. |ECO| replace:: :abbr:`ECO (Engineering change order)`
+.. |ECOs| replace:: :abbr:`ECOs (Engineering change orders)`
+
+Note: any suggestions on a more suitable document name that reflects a specific feature in Odoo? I
+couldn't think of one ):
+
+Easily manage bill of materials (BoM) revision history with :menuselection:`PLM`.  Access previous
+BOM versions, assembly instructions, component details, and past product design files. |BOM|
+effective dates are recorded for error tracking. Changes are compartmentalized in an
+:ref:`engineering change order <plm/eco>`, which makes merging |ECOs| back to the production |BOM|
+simple.
+
+Current BoM version
+===================
+
+To see the current version of the |BOM| used in production, go to :menuselection:`PLM app --> Master
+Data --> Bill of Materials`. Select the desired |BOM| from the list. Then, switch to the
+:guilabel:`Miscellaneous` tab, where the currently live :guilabel:`Version` of the |BOM| is
+displayed.
+
+.. note::
+   |BOMs| can also be accessed from :menuselection:`Manufacturing app --> Products --> Bill of
+   Materials`.
+
+Version history
+===============
+
+To view all |ECOs| linked to a |BOM|, click the :guilabel:`ECO` smart button on a |BOM|. Click the
+:guilabel:`≣ (four horizontal lines)` icon to switch to list view.
+
+.. image:: version_control/eco-smart-button.png
+   :align: center
+   :alt: Show ECO smart button on a BoM.
+
+In the list of |ECOs|, navigate to the search bar and click the :guilabel:`▼ (down arrow)` icon to
+access a drop-down menu of :guilabel:`Filters`. Next, filter by :guilabel:`Done` |ECOs| to view the
+revision history of the |BOM|, the :guilabel:`Responsible` who applied the change, and the
+:guilabel:`Effective Date` of the |BOM|. Click each |ECO| to view the past components, operations,
+and design files associated with the |BOM|.
+
+.. image:: version_control/eco-list.png
+   :align: center
+   :alt: Display ECO revision history for a BoM for a product.
+
+.. tip::
+   If no date is displayed in the :guilabel:`Effective Date` field, this means the
+   :guilabel:`Effective` date of the |ECO| was set to :guilabel:`As soon as possible`.
+   Alternatively, check when the |BOM| went live by navigating to the chatter and hovering over the
+   time the |ECO| was moved to the :ref:`closing stage <plm/eco/stage-config>`.
+
+   .. image:: version_control/effective-asap.png
+      :align: center
+      :alt: Show *Effective* as soon as possible field.
+
+
+Design files
+============
+
+Attach CAD files, PDFs, images, or other design material to the |BOM| itself. To do so, navigate to
```

Comment:
Could be helpful to establish what CAD stands for for the readers on first mention (even though most reading this will probably know :-))